guardrails/actions/filter.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Dict, List
+
+
+class Filter:
+    pass
+
+
+def apply_filters(value: Any) -> Any:
+    if isinstance(value, Filter):
+        pass
```

Comment:
Not exactly, this method is recursive so `Filter` is used to nullify specific values.  For string output that's the whole thing making it no different from `Refrain`, but for structured output it can be just a specific key-value pair.